RELEASES.md

```diff
@@ -57,6 +58,110 @@ Get acquainted with the latest EDP releases.
+
+We have updated terminology within the platform. Environments are now referred to as Deployment Flows, and Stages are now Environments. These changes align our terminology with globally recognized terms, reducing user confusion. For more details, please refer to our updated [Glossary](https://docs.kuberocketci.io/docs/next/glossary).
+
+We have added the ability to specify custom build and review pipelines with just one click when editing or creating new codebase branches. This enhancement increases flexibility when working with codebases, allowing for easier modifications to pipeline logic based on specific requirements.
+
+Another significant enhancement is the introduction of a new section called Pipelines. This section centralizes pipeline observability by consolidating all pipelines in one place. We are continuing to migrate Tekton functionality into our platform. Consequently, we have consolidated our pipelines into a single location. The new Pipelines section displays the entire Tekton Pipeline Run history, simplifying pipeline management and observability.
+
+Users can now deliver their applications in the form of RPM packages, implemented using Ansible. Correspondingly, support for C and C++ languages with make and CMake build tools has been added to the platform.
+
+We have also introduced the ability to [customize Environment deletion](https://docs.kuberocketci.io/docs/next/operator-guide/cd/customize-environment-deletion). This feature is particularly useful for users with applications that have specific configurations or dependencies, e.g. database cleanup, infrastructure tearing down, etc. Customized clean flows make it easier to maintain components throughout their lifecycle.
+
+Additionally, we have added new functionality to our deployment pipelines. A new tab called Variables has been added to the Environment (formerly Stage) details page. In this tab, users can specify key-value parameters, which will be transmitted within the application as environment variables. This improvement allows users to configure application deployment settings without leaving the KubeRocketCI portal.
+
+Deploy pipelines now support manual approval. When creating a custom deploy pipeline, users can include a checkpoint to prompt for approval or rejection before proceeding to the next step.
+
+We have added a new deployment trigger type called "[Auto-stable](https://github.com/epam/edp-cd-pipeline-operator/blob/master/deploy-templates/crds/v2.edp.epam.com_stages.yaml#L161)" has been added to the platform. This trigger type allows for creating an ordered queue of deploy Pipeline Runs, which will be launched sequentially once the previous one finishes. Unlike the "Auto" trigger type, where intermediate pipelines are skipped, "Auto-stable" ensures that each version is deployed, making it easier to identify the exact application version that caused errors during deployment.
```

Comment:
deployment strategy

Reply:
original: where intermediate pipelines are skipped

smth like: where intermediate versions are skipped and the latest versions are deployed